src/components/charts/BarChart.tsx

```diff
@@ -402,33 +427,56 @@ const BarChart = () => {
                       <CustomImage src={refreshIcon} />
                     </CustomButton>
                     <CustomButton
-                      className="home-expand-btn ml-3 pb-[4px]"
+                      className="home-expand-btn ml-3"
                       onClick={handleExpandClick}
                     >
                       <CustomImage src={openNewPageIcon} />
                     </CustomButton>
                   </div>
                 </div>
+                <div className="flex justify-start mb-2 md:mb-0 md:justify-center items-center">
+                  <CustomTab
+                    className="custom-tab md:absolute md:top-[1.25rem] md:right-[8rem]"
+                    tabData={SESSIONS_TABS}
+                    tabValue={tabValue}
+                    setTabValue={setTabValue}
+                  />
+                </div>
+              </>
+            )}
+            {location.pathname.includes("sessions") && (
+              <>
+                <div className="block sm:hidden session-page-title mb-2">
+                  {SESSIONS}
+                </div>
                 <CustomTab
-                  className="custom-tab sm:absolute sm:top-[1.25rem] sm:right-[8rem]"
+                  className={`custom-tab ${width < 640 && "!self-start"}`}
```

Comment:
` className={`custom-tab ${width < 640 ? "!self-start"} : ""`}`
Other wise, it will add `false` to the className

Reply:
done